### ISSUE TITLE
aws-ios-sdk-2.12.6 -- Could not find module 'AWSMobileClient' for target 'x86_64-apple-ios-simulator'; found: arm64, armv7-apple-ios, arm64-apple-ios, arm, armv7

### ISSUE THREAD

**Issue by backslash-f** [aws-amplify/aws-sdk-ios#2274]
Disclaimer: I'm not using Carthage / Cocoapods.

Download the Frameworks from https://aws-amplify.github.io/docs/ios/manualsetup (under "_Download the latest SDK_"). That would be `aws-ios-sdk-2.12.6` by the time I'm writing this.

Build against a simulator. The error will be:
```
Could not find module 'AWSMobileClient' for target 'x86_64-apple-ios-simulator';
found: arm64, armv7-apple-ios, arm64-apple-ios, arm, armv7
```

**Xcode 11.3.1 (11C504)**

(Also please check [this related SO post](https://stackoverflow.com/questions/58873371/how-to-get-aws-frameworks-working-in-xcode-11).)

TL/DR: Is there currently any way to integrate the AWS frameworks without using Carthage or CocoaPods?

And when there would be a new SDK version under the above link?

Thanks.

**Comment by wooj2** [comment]
Hi,

Thank you so much for the bug report here.  I believe I have been able to reproduce the issue you are running into and have submitted a pull request here: https://github.com/aws-amplify/aws-sdk-ios/pull/2281 .

I am optimistically closing this issue, but feel free to re-open if you run into any problems regarding this matter.
